webapp/pages/admin/users.vue

```diff
@@ -21,7 +21,7 @@
     <ds-card v-if="User && User.length">
       <ds-table :data="User" :fields="fields" condensed>
         <template slot="index" slot-scope="scope">
-          {{ scope.row.index }}.
+          {{ scope.row.index + 1 }}.
```

Comment:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2019-08-13T11:28:34Z" title="Tuesday, August 13th 2019, 1:28:34 pm +02:00">Aug 13, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -21,7 +21,7 @@
     <ds-card v-if="User && User.length">
       <ds-table :data="User" :fields="fields" condensed>
         <template slot="index" slot-scope="scope">
-          {{ scope.row.index }}.
+          {{ scope.row.index + 1 }}.
```


:open_mouth: code creep, oh no!!!

Reply:
> [<img alt="Tirokk" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Tirokk) **Authored by [Tirokk](https://github.com/Tirokk)**
_<time datetime="2019-08-14T06:16:17Z" title="Wednesday, August 14th 2019, 8:16:17 am +02:00">Aug 14, 2019</time>_

---


Is this a code creep? @mattwr18